### PR TITLE
Feature/1791 add discernible text label to author twitter links

### LINKED ISSUE #1791: Add discernible text label to author twitter links
https://github.com/INN/largo/blob/512da701664b329f2f92244bbe54880a6e146431/inc/byline_class.php#L157

This link needs something like a `title="follow $person on Twitter"` or `aria-label="follow $person on Twitter"` element.

And this bit too:

https://github.com/INN/largo/blob/512da701664b329f2f92244bbe54880a6e146431/partials/author-bio-social-links.php#L30-L34

This issue was identified during research on https://github.com/INN/largo/issues/1653 using the AXE browser plugin.

### PR DESCRIPTION
**Pull request by vanduc1102**
## Changes

This pull request makes the following changes:

- Add title for links as buttons

## Why

Title for links are missing

For #1791

## Testing/Questions

Features that this PR affects:

- sharing buttons

#### BioPage
Screen Shot 2019-10-08 at 11 11 31 PM

#### inline author

Screen Shot 2019-10-08 at 11 07 53 PM

#### share button 
Screen Shot 2019-10-08 at 11 05 47 PM

INN Member/Labs Client requesting: (if applicable)

- [x ] Contributor has read INN's [GitHub code of conduct](https://github.com/INN/.github/blob/master/CODE_OF_CONDUCT.md)
- [ x] Contributor would like to be mentioned in the release notes as: (fill in this blank)
- [ x] Contributor agrees to the license terms of this repository.

### CODE REVIEW

File: partials/author-bio-social-links.php [hunk 1]
```diff
@@ -23,25 +23,25 @@
...
 		<li class="facebook">
-			<a href="https://facebook.com/<?php echo largo_fb_url_to_username(esc_url( $fb )); ?>" title="<?php echo esc_attr( $author_obj->display_name ); ?> on Facebook" rel="me"><i class="icon-facebook"></i></a>
+			<a href="https://facebook.com/<?php echo largo_fb_url_to_username(esc_url( $fb )); ?>" title="Follow <?php echo esc_attr( $author_obj->display_name ); ?> on Facebook" rel="me"><i class="icon-facebook"></i></a>
 		</li>
 	<?php } ?>
 
 	<?php if ( $twitter = $author_obj->twitter ) { ?>
 		<li class="twitter">
-			<a href="https://twitter.com/<?php echo largo_twitter_url_to_username ( $twitter ); ?>"><i class="icon-twitter"></i></a>
+			<a href="https://twitter.com/<?php echo largo_twitter_url_to_username ( $twitter ); ?>" title="Follow <?php echo largo_twitter_url_to_username ( $twitter ); ?> on Twitter" ><i class="icon-twitter"></i></a>
```
Comment (benlk) [inline partials/author-bio-social-links.php]:
```diff
- echo largo_twitter_url_to_username ( $twitter ); 
+ echo esc_attr( largo_twitter_url_to_username( $twitter ) ); 
```
The twitter username output needs to be passed through `esc_attr` for safety, and PHP does not permit spaces between a function name and the following `(`

File: partials/author-bio-social-links.php [hunk 2]
```diff
@@ -23,25 +23,25 @@
...
 		</li>
 	<?php } ?>
 
 	<?php if ( $email && $show_email !== 'off' ) { ?>
 		<li class="email">
-			<a class="email" href="mailto:<?php echo esc_attr( $email ); ?>" title="e-mail <?php echo esc_attr( $author_obj->display_name ); ?>"><i class="icon-mail"></i></a>
+			<a class="email" href="mailto:<?php echo esc_attr( $email ); ?>" title="E-mail <?php echo esc_attr( $author_obj->display_name ); ?>"><i class="icon-mail"></i></a>
```
Comment (benlk) [inline partials/author-bio-social-links.php]: While we're making changes to this, would you mind removing the `-` in `E-mail`? The dash is somewhat dated.

**Review by benlk — CHANGES_REQUESTED** [review]

**Comment by benlk** [comment]
Thanks for making this pull request! There's some small changes that need to be made, and then we'll be good to merge.

How should we credit you in the changelog? Is `Github user vanduc1102` acceptable, or would you prefer we use a different name?

**Comment by vanduc1102** [comment]
> Thanks for making this pull request! There's some small changes that need to be made, and then we'll be good to merge.
> 
> How should we credit you in the changelog? Is `Github user vanduc1102` acceptable, or would you prefer we use a different name?

thanks @benlk , @vanduc1102  is my preferred username

**Comment by vanduc1102** [comment]
Hi @benlk , comments are addressed

File: partials/author-bio-social-links.php (hunk 1 above)
Comment (vanduc1102) [reply partials/author-bio-social-links.php]: Hi @benlk , comments are addressed

File: partials/author-bio-social-links.php (hunk 2 above)
Comment (vanduc1102) [reply partials/author-bio-social-links.php]: Hi @benlk , comment is addressed

**Comment by joshdarby** [comment]
@vanduc1102 Are you still interested in contributing this PR?

**Comment by vanduc1102** [comment]
hi @joshdarby , yes, i will resolve the conflicts and push

**Comment by vanduc1102** [comment]
Hi @joshdarby , i just updated the PR, please verify and merge if it is possible, thanks.

**Pull request merged**